### DISCUSSION TITLE
Specialized `back-to-indentation` for Magit

### DISCUSSION THREAD

**Discussion by federicotdn** [magit/magit#5183] [Ideas]
Hi! I've been using Magit for some years now, and every now and then I realize I want to copy/kill-as-copy a very specific part of a diff. In general this happens in the Status buffer (`magit-status-mode`) but also also when looking at a specific revision (`magit-revision-mode`). The only very small annoyance is that, since all diff lines start with either `-` or `+`, the `back-to-indentation` command stops being useful, which is unfortunate since I use it quite a lot to move around inside buffers.

I was thinking that maybe Magit could have a `magit-back-to-indentation` command that behaves exactly like the original, but it ignores a single leading `-` or `+` in a line when invoked, Like so:

("|" represents the point)
```diff
+  |      added text
```
`M-x magit-back-to-indentation`
```diff
+        |added text
```

**Comment by tarsius** [comment]
I prefer this idea over #5155,
and have implemented it in 930d34395262af381917800f73a7209c71f45a4f.

**Reply by federicotdn** [reply]
Nice! I wouldn't have mind (minded?) doing it myself 😃 but thank you for adding it! Cheers

**Reply by tarsius** [reply]
You're welcome! :grinning:
